package.json

```diff
@@ -49,20 +49,29 @@
     "graphql": "^16.9.0",
     "husky": "^9.1.6",
     "lint-staged": "^15.2.10",
-    "markprompt": "^0.1.7",
     "prettier": "^3.3.3",
-    "react-markdown": "^9.0.1",
-    "remark-gfm": "^4.0.0",
     "remark-lint-heading-capitalization": "^1.2.0",
     "rimraf": "^6.0.1",
     "uuid": "^10.0.0"
   },
   "dependencies": {
-    "@sentry/react": "^8.42.0",
+    "@sentry/react": "^9.29.0",
     "eslint-plugin-simple-import-sort": "^12.1.1",
-    "lucide-react": "^0.451.0",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-transition-group": "^4.4.5"
+    "lucide-react": "^0.515.0",
+    "react": "^19.0.0",
+    "prism-react-renderer": "^2.4.1",
+    "react-dom": "^19.0.0"
+  },
+  "browserslist": {
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 3 chrome version",
```

Comment:
nitpick: maybe we can have 2 last versions of all browsers - as this means polyfills for Safari 13 (from 2020)